_layouts/posts.html

```diff
@@ -27,15 +27,28 @@
                     <a href="{{ post.url }}"><p>{{ post.title | escape }}</p></a>
                 {% endif %}
                 </div>
-
```

Comment:
Could you also replace in this file the Card creation for each post by  {% include _includes/post.html %}?

Reply:
I tried to do this already but (my) goal was to make the categories page less crowded but doing it would also call the `post.categories` and the `post.excerpt` into the categories page which defeats the less crowded goal
@bebatut what do you think can be done? or should I just go on with it

Reply:
For that, you can add arguments in the ` {% include _includes/post.html %}`, that could be checked in the code of `_includes/post.html`. For example having in the `posts.html` ` {% include _includes/post.html categories=True %}` and in the `_includes/post.html`, check if categories is True to display it